lua/telescope/make_entry.lua

```diff
@@ -240,6 +240,31 @@ do
   end
 end
 
+function make_entry.gen_from_git_stash()
+  local displayer = entry_display.create {
+    separator = " ",
+    items = {
+      { width = 50 },
+      { remaining = true }
+      }
+    }
```

Comment:
You can remove the displayer now as well

Reply:
done